includes/widgets/accordion.php

```diff
@@ -316,7 +316,7 @@ protected function _register_controls() {
 			Group_Control_Typography::get_type(),
 			[
 				'name' => 'title_typography',
-				'selector' => '{{WRAPPER}} .elementor-accordion .elementor-tab-title',
+				'selector' => '{{WRAPPER}} .elementor-accordion .elementor-accordion-title',
```

Comment:
What about backwards compatibility for custom CSS on `elementor-tab-title` selector?

Reply:
You are right. I will bring back that selector in addition

Reply:
@matipojo What kind of backward compatibility is needed here? How can a 3rd party user be affected when we change these selectors?

Reply:
If a user adds Custom CSS like this: 
```
selector .elementor-tab-title {
```
it will stop to work.

Reply:
It WON'T stop to work. please note that he just removed a SELECTOR, not a CLASS